src/prune/tooling/ui.cpp

```
@@ -8,7 +8,51 @@ namespace prune {
+                        m_file_status_is_error = false;
+                    }
+                    else {
+                        m_file_status = "Load failed: " + error;
+                        m_file_status_is_error = true;
+                    }
+                }
+
+                ImGui::Separator();
                 ImGui::MenuItem("Options", nullptr, &m_show_view_grid_options);
+
+                if (!m_file_status.empty()) {
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)